### PR TITLE
[MM-57909] Remove single-handler support for Calls

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

We are fully deprecating this mode in Calls v0.28, shipping with Mattermost v9.10

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-57909

### CODE REVIEW

File: source/configure/calls-deployment.rst [hunk 1]
```diff
@@ -455,4 +446,4 @@ On the server side, run the following:
 
    sudo tcpdump -n port 8443
 
-This command will output information (i.e. source and destination addresses) for all the network packets being sent or received through port ``8443``. This is a good way to check whether data is getting in and out of the instance and can be used to quickly identify network configuration issues.
\ No newline at end of file
+This command will output information (i.e. source and destination addresses) for all the network packets being sent or received through port ``8443``. This is a good way to check whether data is getting in and out of the instance and can be used to quickly identify network configuration issues.
```
Comment (cpoile) [inline source/configure/calls-deployment.rst:449]: Hmm, why is this line marked as modified?

**Review by cpoile — APPROVED** [review]
Thanks!

File: source/configure/calls-deployment.rst (hunk 1 above)
Comment (streamer45) [reply source/configure/calls-deployment.rst:449]: Maybe a new line or white space, not sure.

Comment (cpoile) [reply source/configure/calls-deployment.rst:449]: 🤷

**Pull request merged**